fetch/practica4/index.js

```diff
@@ -1,52 +1,81 @@
-'use strict';
+"use strict";
 
-const BASE_API_URL = 'https://corona.lmao.ninja';
+const BASE_API_URL = "https://corona.lmao.ninja";
 
 async function initMap() {
-    const props = {
-        center: {
-            lat: 15,
-            lng: 0
-        },
-        zoom: 2
-    }
-    const map = new google.maps.Map(document.getElementById('map'), props);
-    let globalData = null;
-    let countriesData = null;
-
-    $('#modalLoading').modal('show');
-    try {
-        const globalRes = await fetch(`${BASE_API_URL}/all`);
-        globalData = await globalRes.json();
-        cacheAPIData('globalData', globalData);
-    }
-    catch (error) {
-        globalData = retrieveCachedAPIData('globalData');
-        if (!globalData) {
-            // TODO
-        }
+  const props = {
+    center: {
+      lat: 15,
+      lng: 0
+    },
+    zoom: 2
+  };
+  const map = new google.maps.Map(document.getElementById("map"), props);
+  let globalData = null;
+  let countriesData = null;
+
+  $("#modalLoading").modal("show");
+  try {
+    const globalRes = await fetch(`${BASE_API_URL}/all`);
+    globalData = await globalRes.json();
+    cacheAPIData("globalData", globalData);
+  } catch (error) {
+    globalData = retrieveCachedAPIData("globalData");
+    if (!globalData) {
+      // TODO
     }
+  }
 
-    try {
-        const countriesRes = await fetch(`${BASE_API_URL}/countries`);
-        countriesData = await countriesRes.json();
-        cacheAPIData('countryData', countriesData);
+  try {
+    const globalRes = await fetch(`${BASE_API_URL}/all`);
+    globalData = await globalRes.json();
+    cacheAPIData("globalData", globalData);
+  } catch (error) {
+    globalData = retrieveCachedAPIData("globalData");
+    if (!globalData) {
+      // TODO
     }
-    catch (error) {
-        countriesData = retrieveCachedAPIData('countryData');
-        if (!countriesData) {
-            // TODO
-        }
+  }
+
+  try {
+    const countriesRes = await fetch(`${BASE_API_URL}/countries`);
+    countriesData = await countriesRes.json();
+    cacheAPIData("countryData", countriesData);
+  } catch (error) {
+    countriesData = retrieveCachedAPIData("countryData");
+    if (!countriesData) {
+      // TODO
     }
```

Comment:
Code from line 40 to 48 is repeated, see lines 29 to 37. Is the same code fragment